charts/home-assistant/ci/ct-values.yaml

```diff
@@ -0,0 +1,2 @@
+ingress:
+  enabled: true
```

Comment:
Maybe we want to run a(n additional) test case where vscode is enabled? Seems to me that is a pretty common scenario that people would want to run.

Reply:
Agreed!

Reply:
How do we add an additional test?

Reply:
> Charts may have multiple custom values files matching the glob pattern '*-values.yaml' in a directory named 'ci' in the root of the chart's directory. The chart is installed and tested for each of these files.

This is according to the `ct` docs, so `vscode-values.yaml` _should_ work